### PR TITLE
[WIP] Add BlazingSQL bindings

### PR DESCRIPTION
**Pull request by trxcllnt**
Opening a draft PR to add BlazingSQL bindings scaffolding. More to come.

### CODE REVIEW

File: modules/blazingsql/CMakeLists.txt [hunk 1]
```diff
@@ -0,0 +1,117 @@
+#=============================================================================
+# Copyright (c) 2020, NVIDIA CORPORATION.
```
Comment (mzegar) [inline modules/blazingsql/CMakeLists.txt:2]: 2021 here

**Review by mzegar — COMMENTED** [review]
probably the only insightful comment I can make on this haha

will try to build this sometime today.

File: modules/blazingsql/CMakeLists.txt (hunk 1 above)
Comment (trxcllnt) [reply modules/blazingsql/CMakeLists.txt:2]:
```suggestion
# Copyright (c) 2021, NVIDIA CORPORATION.
```

File: modules/blazingsql/jest.config.js [hunk 2]
```diff
@@ -0,0 +1,60 @@
+// Copyright (c) 2020-2021, NVIDIA CORPORATION.
```
Comment (trxcllnt) [inline modules/blazingsql/jest.config.js:1]:
```suggestion
// Copyright (c) 2021, NVIDIA CORPORATION.
```

File: modules/core/cmake/Modules/ConfigureBlazingSQL.cmake [hunk 3]
```diff
@@ -0,0 +1,116 @@
+#=============================================================================
+# Copyright (c) 2020-2021, NVIDIA CORPORATION.
```
Comment (trxcllnt) [inline modules/core/cmake/Modules/ConfigureBlazingSQL.cmake:2]:
```suggestion
# Copyright (c) 2021, NVIDIA CORPORATION.
```

**Pull request merged**